src/main/resources/application.yml

```diff
@@ -30,24 +30,6 @@ spring:
             client-id: ${basm.client.client-id}
             client-secret: ${basm.client.client-secret}
             authorization-grant-type: client_credentials
-          prison-api:
-            provider: hmpps-auth
-            client-id: ${system.client.id}
-            client-secret: ${system.client.secret}
-            authorization-grant-type: client_credentials
-            scope: read
-          hmpps-auth-api:
-            provider: hmpps-auth
-            client-id: ${system.client.id}
-            client-secret: ${system.client.secret}
-            authorization-grant-type: client_credentials
-            scope: read
-          prisoner-search-api:
-            provider: hmpps-auth
-            client-id: ${system.client.id}
-            client-secret: ${system.client.secret}
-            authorization-grant-type: client_credentials
-            scope: read
         provider:
           hmpps-auth:
```

Comment:
Could remove this too - nothing is relying on this 'provider' any more?

Reply:
thanks, yep - will do this as a follow up